apps/hashdotdev/src/pages/roadmap/technology-tree.tsx

```diff
@@ -0,0 +1,423 @@
+                  "linear-gradient(90deg, rgba(0, 141, 185, 0.20) 0%, #5DBEDC 20.83%, #DAF8FF 84.90%, rgba(218, 248, 255, 0.20) 100%)",
+              }}
+            />
+            {nodes.length === 0 ? (
+              <Box
+                sx={{
+                  position: "absolute",
+                  left: "50%",
+                  top: "50%",
+                  transform: "translate(-50%, -50%)",
+                  zIndex: 1,
+                  textAlign: "center",
+                }}
+              >
+                <FaIcon
+                  name="warning"
+                  type="light"
+                  sx={{
+                    fontSize: 42,
+                    color: ({ palette }) => palette.gray[50],
+                    marginBottom: 1,
+                  }}
+                />
+                <Typography
+                  sx={{
+                    fontSize: 15,
+                    color: ({ palette }) => palette.gray[90],
+                    fontWeight: 500,
+                  }}
+                >
+                  No roadmap items match your current filters
+                </Typography>
+                {displayedStatuses.length === 0 ||
+                displayedVariants.length === 0 ||
+                displayedUseCases.length === 0 ? (
+                  <Typography
+                    sx={{
+                      fontSize: 14,
+                      color: ({ palette }) => palette.gray[70],
+                      fontWeight: 400,
+                    }}
+                  >
+                    Please select at least one{" "}
+                    <strong>
+                      {displayedStatuses.length === 0
+                        ? "status"
+                        : displayedVariants.length === 0
+                        ? "type"
+                        : "use case"}
+                    </strong>{" "}
+                    to view matching deliverables
+                  </Typography>
+                ) : null}
+              </Box>
+            ) : null}
+            <Box
+              ref={graphWrapperRef}
+              sx={{
+                position: "relative",
+                width: "100%",
+                overflow: "hidden",
+                height: "50vh",
+                borderBottomColor: ({ palette }) => palette.gray[20],
+                borderBottomStyle: "solid",
+                borderBottomWidth: 1,
+              }}
+            >
+              <Box ref={graphRef}>
+                <Box
+                  sx={{
+                    position: "relative",
+                    zIndex: 1,
+                  }}
+                >
+                  {nodes.map(({ x, y, data }) => (
+                    <TechnologyTreeNode
+                      key={data.id}
+                      x={x}
+                      y={y}
+                      data={data}
+                      blurred={blurredNodes.includes(data.id)}
+                      onHover={() => setFocusedNodeId(data.id)}
```

Comment:
Maybe we can make this an `onClick` instead or consider some other means of selecting/deselecting nodes? It's hard to navigate while keeping the info tooltip hovered. cc @nonparibus

Reply:
Happy to sub out this behavior. It's not clear to me it's actually useful in its current form.

Reply:
Happy to do this, but do we still want to tooltip to display on hover? Or on click as well?

Reply:
I would lean towards onclick, because if the tooltip appears on hover there is nothing to indicate that something would happen if you clicked on it. Or doing something else entirely to make the paths highlighted

Reply:
Alternatively we could have some other icon for highlight, or make the entire node clickable (appreciate we want to be able to link to 'read more' also, but there could be a smaller icon/link within the node for that)

Reply:
I've made it on click in https://github.com/hashintel/hash/pull/2747/commits/2f3abe141d23882d12a8d535b242a4a29f385b7a, but think a separate button might be more appropriate. There's also a bug where if you then zoom, the tooltip becomes misaligned with the node which may not be very straightforward to address.

Reply:
I'd rather not introduce a new button, as it'll add clutter to an already crowded interface.

I think we can afford for this functionality to be slightly hard to discover. Its utility in the first place remains unclear to me.

Reply:
So to be clear you want me make it on hover again @nonparibus?

Reply:
What? No. Leave as onClick. I'm just saying don't add another icon that adds extra clutter.